src/routes/safe/components/Settings/SpendingLimit/NewLimitModal/Review.tsx

```diff
@@ -67,6 +68,10 @@ const useExistentSpendingLimit = ({
   }, [spendingLimits, txToken.decimals, values.beneficiary, values.token])
 }
 
+const previousResetTime = (existentSpendingLimit: SpendingLimit) =>
+  getResetTimeOptions().find(({ value }) => value === (+existentSpendingLimit.resetTimeMin).toString())?.label ??
```

Comment:
Why convert this from a string to a number and back to a string? Can you not select the label directly?

Reply:
not my code ser